compiler/include/symbol.h

```diff
@@ -462,6 +462,8 @@ class ModuleSymbol : public Symbol {
   void                 moduleUseAdd(ModuleSymbol* module);
   void                 moduleUseRemove(ModuleSymbol* module);
 
+  bool                 isVisible(BaseAST* scope);
```

Comment:
As I understand things, adding "const" to a member function definition merely documents that
this method does not perform any side-effects on instance variables.  This in turn means it
would be valid to do

const ModuleSymbol* foo = <...>;

foo->isVisible(scope);

It doesn't have anything to do with memorizing etc.

Mike